fenix/app/src/main/java/org/mozilla/fenix/settings/quicksettings/protections/cookiebanners/CookieBannerDetailsController.kt

```diff
@@ -72,31 +82,29 @@ class DefaultCookieBannerDetailsController(
     override fun handleBackPressed() {
         getCurrentTab()?.let { tab ->
             context.components.useCases.trackingProtectionUseCases.containsException(tab.id) { contains ->
-                ioScope.launch {
-                    val hasException = if (context.settings().shouldUseCookieBanner) {
-                        cookieBannersStorage.hasException(tab.content.url, tab.content.private)
-                    } else {
-                        false
-                    }
-                    withContext(Dispatchers.Main) {
-                        fragment.runIfFragmentIsAttached {
-                            navController().popBackStack()
-                            val isTrackingProtectionEnabled = tab.trackingProtection.enabled && !contains
-                            val directions =
-                                BrowserFragmentDirections.actionGlobalQuickSettingsSheetDialogFragment(
-                                    sessionId = tab.id,
-                                    url = tab.content.url,
-                                    title = tab.content.title,
-                                    isSecured = tab.content.securityInfo.secure,
-                                    sitePermissions = sitePermissions,
-                                    gravity = gravity,
-                                    certificateName = tab.content.securityInfo.issuer,
-                                    permissionHighlights = tab.content.permissionHighlights,
-                                    isTrackingProtectionEnabled = isTrackingProtectionEnabled,
-                                    isCookieHandlingEnabled = !hasException,
-                                )
-                            navController().navigate(directions)
-                        }
+                ioScope.launch(Dispatchers.Main) {
```

Comment:
Let's also change this as we did with the other ones :)

Reply:
Done